buildmaster/LHCB_DY_13TEV_DIELECTRON/metadata.yaml

```diff
@@ -0,0 +1,53 @@
+# Previously called `LHCB_Z_13TEV_DIELECTRON`
+# author: "Tanjona R. Rabemananjara"
+setname: "LHCB_DY_13TEV_DIELECTRON"
+
+version: 1
+version_comment: "First implementation"
+
+# References
+arXiv:
+  url: "https://arxiv.org/abs/1607.06495"
+iNSPIRE:
+  url: "https://inspirehep.net/literature/1477581"
+hepdata:
+  url: "https://www.hepdata.net/record/ins1477581"
+  version: 1
+
+nnpdf_metadata:
+  nnpdf31_process: "DY NC"
+  experiment: "LHCb"
+
+implemented_observables:
+- observable_name: "y"
+  observable:
+    description: "Differential cross-section of Z-->ee as a function of Z-rapidity"
+    label: r"$d\isgma / d|y|$"
+    units: "[fb]"
+  process_type: "DY"
+  ndata: 17
+  tables: [5]
+  npoints: [17]
+
+  # Plotting information
+  plotting:
+      dataset_label: "LHCb $Z\\to ee$"
+      plot_x: y
+      y_label: '$d\sigma_{Z}/dy$ (fb)'
```

Comment:
This is a tricky one and related to the comment above, after finding the `process_type` you will need to look at the `PLOTTINGTYPE_<process_type>` file as well. In this case:

https://github.com/NNPDF/nnpdf/blob/master/nnpdfcpp/data/commondata/PLOTTINGTYPE_EWK_RAP.yaml

So the plotting is missing the kinematic override: `ewk_rap_sqrt_scale`

The kinematic override is then used to prepare the plots.

Btw, not mandatory, but if you write the list in `kinematic_coverage` in the same order as in https://github.com/NNPDF/nnpdf/blob/94917a4bdddc5e99f1835d61e7a0540f8ef07052/validphys2/src/validphys/filters.py#L19 things are more likely to work out of the box.
Of course, the idea is that (eventually) the order is not considered anymore, just the names, but for that we first need to have many datasets implemented so that we can enforce some common naming for the variables.

Reply:
They are ordered in the same way but the 1st and 3rd variable are named differently, should we change them from `y` to `etay` and `sqrts` to `sqrt_s` to conform with `"EWK_RAP": ("etay", "M2", "sqrts")`?

Reply:
No, just the order is fine (and not even necessary, just making the success a bit more likely, of course, same name even greater chance). The standardization of the naming I would leave for the end.